src/Nethermind/Nethermind.TxPool/TxBroadcaster.cs

```diff
@@ -117,6 +124,13 @@ public void BroadcastOnce(ITxPoolPeer peer, Transaction[] txs)
 
         public void BroadcastPersistentTxs()
         {
+            if (_lastPersistedTxBroadcast + _minTimeBetweenPersistedTxBroadcast > DateTimeOffset.Now)
```

Comment:
very minor put `DateTimeOffset.Now` to local variable instead of accessing it 2 times.